test/pipe.js

```diff
@@ -76,8 +107,6 @@ if (process.platform !== 'win32') {
     });
   } else {
     console.error('Warning: Cannot verify piped exec');
+    shell.exit(123);
```

Comment:
Why was this added? It looks like it should've been here already. Did something change here, or did we just forget this before?